CHANGES.rst

```diff
@@ -9,6 +9,8 @@
 
 - Implemented support and tests for linearity reference model. Corrected dimension order in factories. Added primary array definition to MaskRefModel. [#39]
 
+- Changed exposure ``start_time``, ``mid_time``, and ``end_time`` to string to match RAD update. [#40]
+
```

Comment:
I might be good to include the format for the string,  
string is ISO 8601 format with 'T' as a separator.

Reply:
Well, this isn't enforced in RDM (and in one test, just a generic string is used). This would be specified in RAD.